Viper Practice/src/entity/Fruit.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Fruit.swift
+//  Viper Practice
+//
+//  Created by ahsan shabir on 02/04/2024.
+//
+
+import Foundation
+
+struct Fruit: Codable {
+
+  var name: String? = nil
+  var id: Int? = nil
+  var family: String? = nil
+  var order: String? = nil
+  var genus: String? = nil
+  var nutritions: Nutritions? = Nutritions()
+
+  enum CodingKeys: String, CodingKey {
```

Comment:
I think we don't need coding keys if key name is same

Reply:
coding keys removed